README.md

```diff
@@ -15,6 +15,12 @@ To see Codalab Competition's in action, visit [www.competitions.codalab.org](htt
 The CodaLab community forum is hosted on Google Groups.
 - [CodaLabDev Google Groups Forum](https://groups.google.com/forum/#!forum/codalabdev)
 
+## Tests
+
```

Comment:
I never touch this file, can it be because I started this branch from main instead of develop?